subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5_ppi.h

```diff
@@ -369,7 +369,15 @@ static inline void hal_sw_switch_timer_clear_ppi_config(void)
 /* Wire the RADIO EVENTS_END event to one of the PPI GROUP TASK ENABLE task.
  * 2 adjacent PPI groups are used for this wiring. 'index' must be 0 or 1.
  */
+#if defined(CONFIG_SOC_NRF52805)
+#if (EVENT_TIMER_ID == 0)
+#define HAL_SW_SWITCH_GROUP_TASK_ENABLE_PPI 2
+#else
+#error "Insufficient resources"
```

Comment:
I guess you should explain the reason for the "insufficient resources", either in a comment, or directly in the preprocessor #error 

Hard to see how the insufficient resources relates to the ID of the Event TImer.

Reply:
I'll add some more descriptive message/comment.

Reply:
Added.